### PR TITLE
Final Review: Full JavaScript WebApp with PokeAPi and InvolvementAPI

### PR DESCRIPTION
**Pull request by sneeu-leeu**
## In this Project I:
- Created a Webapp with Webpacker 
-  Used the PokeAPI as well as the Involvement API to GET and POST data
- Built a system where you can like the pokemon through the involvement API
- Wrote testing for the item count on the homepage
- Used ES6

PLEASE NOTE> My partner was removed from the project so ONLY STUDENT A applies to this project!

Thanks, enjoy!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<body>
+  <header>
+    <nav id='nav-anchor'>
+     
+    </nav>
+  </header>
+
+  <main>
+    <div id='Pokemon-display'></div>
+    <div id='comment-display'></div>
```
Comment (abruzy) [inline src/index.html:23]: - [ ] when comment button is click, comment modal didn't pop up, this is the the error I get from the console
Screenshot 2021-08-14 at 20 45 18

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu & @alexander16108 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_1. Kindly add the link to the recorded video presenting the features of your project in your PR summary, you can use either zoom or loom._
_2. Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sneeu-leeu** [comment]
@abruzy please note that anything regarding the comment section MUST BE IGNORED as my partner wgo was in charge of that was just removed from the project by student success. Only the pokeAPI and Likes from involvement API must be reviwed

**Comment by SpaYco** [comment]
Hello 👋,

_This is an additional review as a result of the second opinion request._

You're right, only Student A work should be done here since your partner got removed, please do the other requirement (`Kindly add the link to the recorded video presenting the features of your project in your PR summary, you can use either zoom or loom.`) then ask for another review and if the change is done correctly, it would be approved :)


Keep rocking 🚀!

**Comment by sneeu-leeu** [comment]
Hi Aziz,

thanks for the feedback, but I do have a loom video and even my rereview
was bad.

**Pull request merged**